semeio/jobs/correlated_observations_scaling/job_config.py

```diff
@@ -44,10 +44,10 @@ def _realize_list(input_string):
     """
     real_list = []
     for elem in input_string.split(","):
+        if elem.startswith("-"):
+            raise ValueError("Do no accept negative elements")
```

Comment:
Might want to reword this message a bit.

Reply:
Rephrased. Also note that there will be two errors thrown, and this is only from the transformation. The error from validation will still be output.